trace_point.rb

```diff
@@ -1,95 +1,93 @@
 #
-#	    trace = TracePoint.new(:raise) do |tp|
-#		p [tp.lineno, tp.event, tp.raised_exception]
-#	    end
-#	    #=> #<TracePoint:disabled>
+#     trace = TracePoint.new(:raise) do |tp|
+#       p [tp.lineno, tp.event, tp.raised_exception]
+#     end
+#     #=> #<TracePoint:disabled>
 #
-#	    trace.enable
-#	    #=> false
+#     trace.enable  #=> false
 #
-#	    0 / 0
-#	    #=> [5, :raise, #<ZeroDivisionError: divided by 0>]
+#     0 / 0
+#     #=> [5, :raise, #<ZeroDivisionError: divided by 0>]
 #
 # == Events
 #
-# If you don't specify the type of events you want to listen for,
+# If you don't specify the types of events you want to listen for,
 # TracePoint will include all available events.
 #
-# *Note* do not depend on current event set, as this list is subject to
-# change. Instead, it is recommended you specify the type of events you
+# *Note:* Do not depend on the current event set, as this list is subject to
+# change. Instead, it is recommended to specify the types of events you
 # want to use.
 #
 # To filter what is traced, you can pass any of the following as +events+:
 #
-# +:line+:: execute an expression or statement on a new line
-# +:class+:: start a class or module definition
-# +:end+:: finish a class or module definition
-# +:call+:: call a Ruby method
-# +:return+:: return from a Ruby method
-# +:c_call+:: call a C-language routine
-# +:c_return+:: return from a C-language routine
-# +:raise+:: raise an exception
-# +:rescue+:: rescue an exception
-# +:b_call+:: event hook at block entry
-# +:b_return+:: event hook at block ending
-# +:a_call+:: event hook at all calls (+call+, +b_call+, and +c_call+)
-# +:a_return+:: event hook at all returns (+return+, +b_return+, and +c_return+)
-# +:thread_begin+:: event hook at thread beginning
-# +:thread_end+:: event hook at thread ending
-# +:fiber_switch+:: event hook at fiber switch
-# +:script_compiled+:: new Ruby code compiled (with +eval+, +load+ or +require+)
+# +:line+:: Execute an expression or statement on a new line.
+# +:class+:: Start a class or module definition.
+# +:end+:: Finish a class or module definition.
+# +:call+:: Call a Ruby method.
+# +:return+:: Return from a Ruby method.
+# +:c_call+:: Call a C-language routine.
+# +:c_return+:: Return from a C-language routine.
+# +:raise+:: Raise an exception.
+# +:rescue+:: Rescue an exception.
+# +:b_call+:: Event hook at block entry.
+# +:b_return+:: Event hook at block ending.
+# +:a_call+:: Event hook at all calls (+call+, +b_call+, and +c_call+).
+# +:a_return+:: Event hook at all returns (+return+, +b_return+, and +c_return+).
+# +:thread_begin+:: Event hook at thread beginning.
+# +:thread_end+:: Event hook at thread ending.
+# +:fiber_switch+:: Event hook at fiber switch.
+# +:script_compiled+:: New Ruby code compiled (with +eval+, +load+, or +require+).
 #
 class TracePoint
   # call-seq:
-  #	TracePoint.new(*events) { |obj| block }	    -> obj
+  #   TracePoint.new(*events) { |tp| block } -> obj
```

Comment:
I think the returned object is the same one as the one passed into the block.

```suggestion
  #   TracePoint.new(*events) { |tp| block } -> tp
```